cli/env/env.go

```diff
@@ -0,0 +1,23 @@
+package env
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/tarantool/tt/cli/config"
+)
+
+// CreateEnvString generates environment variables for 'tarantool' and 'tt' installed using 'tt'.
+func CreateEnvString(cliOpts *config.CliOpts) string {
+	binDir := cliOpts.Env.BinDir
+	path := os.Getenv("PATH")
+	if !strings.Contains(path, binDir) {
+		path += ":" + binDir
+	}
+
+	tarantoolDir := filepath.Join(cliOpts.Env.IncludeDir, "include", "tarantool")
+
+	return fmt.Sprintf("%s=%s\n%s=%s\n", "PATH", path, "TARANTOOL_DIR", tarantoolDir)
```

Comment:
Is it ok to overwrite TARANTOOL_DIR?

Reply:
I will give an example, 
```
better0fdead@better0fdead:~$ which tarantool
better0fdead@better0fdead:~$ tarantool

Command 'tarantool' not found, but can be installed with:

sudo snap install tarantool             # version 1.9~git, or
sudo apt  install tarantool             # version 1.9.1.26.g63eb81e3c-1.1build3
sudo apt  install tarantool-lts-client  # version 1.5.5.37.g1687c02-1build3

See 'snap info tarantool' for additional versions.

better0fdead@better0fdead:~$ cd queue/
better0fdead@better0fdead:~/queue$ cmake .
-- The C compiler identification is GNU 9.4.0
-- The CXX compiler identification is GNU 9.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:146 (message):
  Could NOT find TARANTOOL (missing: TARANTOOL_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake-3.16/Modules/FindPackageHandleStandardArgs.cmake:393 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindTarantool.cmake:26 (find_package_handle_standard_args)
  CMakeLists.txt:11 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/better0fdead/queue/CMakeFiles/CMakeOutput.log".
better0fdead@better0fdead:~/queue$ cd ../tt
better0fdead@better0fdead:~/tt$ . <(./tt env)
better0fdead@better0fdead:~/tt$ cd ..
better0fdead@better0fdead:~$ cd queue/
better0fdead@better0fdead:~/queue$ printenv TARANTOOL_DIR
/home/better0fdead/tt/include/include
better0fdead@better0fdead:~/queue$ cmake .
-- Found TARANTOOL: /home/better0fdead/tt/include/include (found version "2.11.1-0-g96877bd") 
-- Tarantool LUADIR is share/tarantool
-- Tarantool LIBDIR is lib/tarantool
-- Configuring done
-- Generating done
-- Build files have been written to: /home/better0fdead/queue
```

So, in many repo's TARANTOOL_DIR environmental variable is used to find tarantool include files.
Also system tarantool doesn't set TARANTOOL_DIR so we don't overwrite it.
```
better0fdead@better0fdead:~$ sudo apt install tarantool
[sudo] password for better0fdead: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
The following packages were automatically installed and are no longer required:
  gir1.2-goa-1.0 libfwupdplugin1 libxmlb1
Use 'sudo apt autoremove' to remove them.
The following NEW packages will be installed:
  tarantool
0 upgraded, 1 newly installed, 0 to remove and 0 not upgraded.
Need to get 0 B/12,0 MB of archives.
After this operation, 53,9 MB of additional disk space will be used.
Selecting previously unselected package tarantool.
(Reading database ... 201644 files and directories currently installed.)
Preparing to unpack .../tarantool_2.11.1.g96877bd353-1_amd64.deb ...
Unpacking tarantool (2.11.1.g96877bd353-1) ...
Setting up tarantool (2.11.1.g96877bd353-1) ...
update-alternatives: using /usr/bin/tarantool to provide /usr/bin/lua (lua-interpreter) in auto mode
Processing triggers for man-db (2.9.1-1) ...
better0fdead@better0fdead:~$ printenv TARANTOOL_DIR
better0fdead@better0fdead:~$ 
```